docs/medical-api/more-info/user-roles.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "User Roles & Permissions"
+description: "Manage Users and Roles in Metriport"
+---
+
+## Overview
+
+Metriport provides a robust user management system that allows you to control access to your account via the dashboard. This guide will walk you through viewing users, inviting new team members, and understanding the different user roles and their permissions.
+
+## Viewing Users
+
+To view all users in your organization:
+
+1. Navigate to the Users page from the left sidebar
+2. You'll see a table listing all users with
+   their:
+   - Email address
+   - Assigned role
+   - Current status (Active/Pending)
+3. Use the search bar at
+   the top to filter users by email address
+
+## Inviting Users
+
+To invite a new user to your Metriport account:
+
+1. Click the "Invite user" button at the top of the Users page
+2. Enter the user's email address
+3. Select their role from the dropdown menu
+4. Click "Invite"
+
+## User Roles & Permissions
+
+Metriport offers several roles to help you manage access levels appropriately. Each role has specific permissions designed to provide the right level of access for different team members:
+
+### Owner
+
+- Full system access
+- Can manage all users and roles
+- Can access billing and subscription settings
+- Can manage all clinical and technical settings
+
+### Admin
+
+- Similar to the Owner role
+- Can manage all users and roles
+- Can access billing and subscription settings
+- Can manage all clinical and technical settings
+
+### Clinical
+
+- Can create, read, and update patients
+- Can create, read, and update facilities
+- Can view patients' medical history
+- No access to user management, billing, or developer pages
+
+### Billing
+
+- Can only access the billing page
+- Can view invoices and manage payment methods
+- No access to clinical data, user management, or any other pages
+
+### Developer
+
+- Can create, read, and update patients
+- Can create, read, and update facilities
+- Can view patients' medical history
+- Can access the developers tab, including API credentials and webhook settings
+- No access to user management or billing pages
```

Comment:
would a table be easier to understand? ie what we did before with DAPI to denote which data types which data sources supported